.circleci/config.yml

```diff
@@ -8,7 +8,7 @@ jobs:
         working_directory: ~/evaluate
         docker:
             - image: cimg/python:3.7
-        resource_class: medium
+        resource_class: large
```

Comment:
Yes, it is because of the evaluator/trainer. I don't understand why, locally running the tests I was using at most 500 MB of RAM for the evaluator/trainer test. 

With medium: https://app.circleci.com/pipelines/github/huggingface/evaluate/468/workflows/5b57ecc8-abd1-4fdb-a1fb-db655510fc60/jobs/1409/resources
With large: https://app.circleci.com/pipelines/github/huggingface/evaluate/479/workflows/7f837ed5-bb94-4c60-a26d-9ba0bfae3c93/jobs/1442/resources

The models in the parity tests are 45 MB and 18 MB. The "beans" dataset looks to be <200 MB ( https://huggingface.co/datasets/beans/tree/main/data ). sst2 should be < 10 MB. So I am not sure what the issue is.